lib/utils/properties-panel.js

```diff
@@ -447,6 +455,10 @@ export function getErrorMessage(id, report) {
     return 'Output element must be defined.';
   }
 
+  if (id === 'completionCondition' && type === ERROR_TYPES.PROPERTY_NOT_ALLOWED) {
+    return getNotSupportedMessage('', allowedVersion);
+  }
+
```

Comment:
Let's adjust the message for `cancelRemainingInstances` too, just like we do with binding types:
<img width="239" alt="image" src="https://github.com/user-attachments/assets/554636ae-6222-4662-86e2-97973e11664a" />

Reply:
It could also be changed to "Must be checked" as full details are in the linter message.

Reply:
I like the second, concise option - changed it and updated the PR screenshot 👍